docs/intro/icpc.md

```diff
@@ -1,54 +1,54 @@
 author: NachtgeistW
 
-## 赛事介绍
+## Introduction
 
 ### ICPC
 
- **ICPC** （英文：International Collegiate Programming Contest，中文：国际大学生程序设计竞赛）由 ICPC 基金会（英文：ICPC Foundation）举办，是最具影响力的大学生计算机竞赛。由于以前 ACM 赞助这个竞赛，也有很多人习惯叫它 ACM 竞赛。
+ **ICPC**, International Collegiate Programming Contest, is organized by the ICPC Foundation. It is the most influential programming competition for college students. Since ACM sponsored this competition in the past, many people are used to calling it ACM competition.
 
-ICPC 主要分为区域赛（Regionals）和总决赛（World Finals）两部分。
+ICPC is mainly divided into two parts: Regionals and World Finals.
 
-官网地址： <https://icpc.baylor.edu> 
+Official website: <https://icpc.baylor.edu> 
 
 ### CCPC
 
-官网地址： <https://ccpc.io> 
+Official website: <https://ccpc.io> 
 
-中国大学生程序设计竞赛。
+Chinese college student programming competition.
 
-和 ICPC 显著的区别是很多学校是不报销的。
+The major difference from ICPC is that many schools do not support reimbursement.
 
-## 赛制介绍
+## Competition system introduction
 
-选手在比赛时有多次提交机会。比赛实时评测并返回结果（部分比赛可以看到错误的测试样例，如 LeetCode），如果提交的结果错误会有罚时，错误次数越多，加罚的时间也越长。每个题目只有在所有数据点全部正确后才能得到分数。比赛排名根据做题数和罚时来评判。
+Contestants have multiple submission opportunities during the competition. The competition is evaluated in real time and the results are displayed (wrong test cases can be seen in some competitions, such as LeetCode). If the submitted result is wrong, there will be a penalty time. The more errors, the longer the penalty time. Each question can only get a score after all the points are correct. The competition ranking is judged based on the number of questions and penalty time.
```

Comment:
solutions are judged in real time

There will be penalty if wrong solutions are submitted

```suggestion
Contestants have multiple submission opportunities during the competition. The competition is evaluated in real time and the results are displayed (wrong test cases can be seen in some competitions, such as LeetCode). If the submitted result is wrong, there will be a penalty time. The more errors, the longer the penalty time. The score of the problem can only be obtained after the submission is accepted. The ranking is judged based on the number of questions and penalty time.
```